pages/features.html

```diff
@@ -70,8 +70,8 @@ <h2><a class="headline-link" href="#hardware">{% trans "Customizable DJ Hardware
 
     <div class="grid-container halves u-align-left">
       <div>
-        <div style="max-height: 200px;">
-          <img class="responsive drop-shadow" src="{% static '/images/2.0/feature-midi.png' %}">
+        <div>
+          <img class="responsive drop-shadow" src="{% static '/images/2.3/feature-midi.png' %}">
```

Comment:
Why is this file in a versioned directory? It's not a screenshot; it shouldn't change version to version.